src/Panel/TimerPanel.php

```diff
@@ -61,18 +61,18 @@ public function implementedEvents(): array
                 ['priority' => 999, 'callable' => $after('Event: Controller.startup')],
                 ['priority' => 999, 'callable' => function () {
                     DebugMemory::record(__d('debug_kit', 'Controller action start'));
-                    DebugTimer::start(__d('debug_kit', 'Controller action'));
+                    DebugTimer::start(__d('debug_kit', 'Controller: action'));
```

Comment:
Could you remove the translation function usages too please. They don't serve any purpose here.

Reply:
done.